### PR TITLE
Benchmarks fix

### PR DESCRIPTION
**Pull request by AboudyKreidieh**
- imported changes to the baselines from berkeleyflow
- cleanup to the docstring to pass `pydocstyle --convention=numpy`
- tests to ensure the baselines are running. TODO: check the return is within a reasonable range

### CODE REVIEW

File: flow/benchmarks/baselines/grid0.py [hunk 1]
```diff
@@ -1,14 +1,6 @@
-"""
-Script to evaluate the baseline performance of grid1 without RL control
-Baseline is an actuated traffic light provided by SUMO
-
-Grid/green wave example
-
-Action Dimension: (9, )
+"""Evaluates the baseline performance of grid0 without RL control.
```
Comment (eugenevinitsky) [inline flow/benchmarks/baselines/grid0.py]: No s as per docstring

File: flow/benchmarks/baselines/grid1.py [hunk 2]
```diff
@@ -1,14 +1,6 @@
-"""
-Script to evaluate the baseline performance of grid1 without RL control
-Baseline is an actuated traffic light provided by SUMO
-
-Grid/green wave example
-
-Action Dimension: (25, )
+"""Evaluates the baseline performance of grid1 without RL control.
```
Comment (eugenevinitsky) [inline flow/benchmarks/baselines/grid1.py]: No s as per docstring

File: flow/benchmarks/baselines/merge012.py [hunk 3]
```diff
@@ -0,0 +1,114 @@
+"""Evaluates the baseline performance of merge without RL control.
```
Comment (eugenevinitsky) [inline flow/benchmarks/baselines/merge012.py]: No s as per docstring

File: flow/benchmarks/rllab/trpo_runner.py [hunk 4]
```diff
@@ -1,5 +1,4 @@
-"""
-Runner script for environments located in flow/benchmarks.
+"""Runs the environments located in flow/benchmarks.
```
Comment (eugenevinitsky) [inline flow/benchmarks/rllab/trpo_runner.py]: No s as per docstring

File: flow/benchmarks/rllib/ars_runner.py [hunk 5]
```diff
@@ -1,9 +1,10 @@
-"""
-Runner script for environments located in flow/benchmarks.
+"""Runs the environments located in flow/benchmarks.
```
Comment (eugenevinitsky) [inline flow/benchmarks/rllib/ars_runner.py]: No s as per docstring

File: flow/benchmarks/rllib/ars_runner.py [hunk 6]
```diff
@@ -1,9 +1,10 @@
...
 
 The environment file can be modified in the imports to change the environment
-this runner script is executed on. Furthermore, the rllib specific algorithm/
-parameters can be specified here once and used on multiple environments.
+this runner script is executed on. This file runs the ARS algorithm in rllib
+and utilizes the hyper-parameters specified in:
+
+TODO: add paper reference
```
Comment (eugenevinitsky) [inline flow/benchmarks/rllib/ars_runner.py]: Simple random search provides a competitive approach to reinforcement learning
by Mania et. al

File: flow/benchmarks/rllib/es_runner.py [hunk 7]
```diff
@@ -0,0 +1,8 @@
+"""Runs the environments located in flow/benchmarks.
+
+The environment file can be modified in the imports to change the environment
+this runner script is executed on. This file runs the ES algorithm in rllib
+and utilizes the hyper-parameters specified in:
+
+TODO: add paper reference
```
Comment (eugenevinitsky) [inline flow/benchmarks/rllib/es_runner.py]: Evolution Strategies as a Scalable Alternative to Reinforcement Learning
by Salimans et. al.

File: flow/benchmarks/rllib/ppo_runner.py [hunk 8]
```diff
@@ -1,9 +1,10 @@
...
-Runner script for environments located in flow/benchmarks.
+"""Runs the environments located in flow/benchmarks.
 
 The environment file can be modified in the imports to change the environment
-this runner script is executed on. Furthermore, the rllib specific algorithm/
-parameters can be specified here once and used on multiple environments.
+this runner script is executed on. This file runs the PPO algorithm in rllib
+and utilizes the hyper-parameters specified in:
+
+TODO: add paper reference
```
Comment (eugenevinitsky) [inline flow/benchmarks/rllib/ppo_runner.py]: Proximal Policy Optimization Algorithms
by Schulman et. al.

File: tests/slow_tests/test_benchmarks.py [hunk 9]
```diff
@@ -0,0 +1,88 @@
...
+from flow.benchmarks.baselines.grid1 import grid1_baseline
+from flow.benchmarks.baselines.merge012 import merge_baseline
+
+os.environ["TEST_FLAG"] = "True"
+
+
+class TestBaselines(unittest.TestCase):
+
+    """
+    Tests that the baselines in the benchmarks folder are running and
```
Comment (eugenevinitsky) [inline tests/slow_tests/test_benchmarks.py]: no s

File: tests/slow_tests/test_benchmarks.py [hunk 10]
```diff
@@ -0,0 +1,88 @@
...
+    returning expected values (i.e. values that match those in the CoRL paper
+    reported on the website, or other).
+    """
+
+    def test_bottleneck0(self):
+        """
```
Comment (eugenevinitsky) [inline tests/slow_tests/test_benchmarks.py]: Thanks! We needed this.

File: flow/benchmarks/baselines/grid0.py (hunk 1 above)
Comment (AboudyKreidieh) [reply flow/benchmarks/baselines/grid0.py]: this only applies to functions. classes and the start of a file do not need to follow the imperative convention (same applies for all below comments).

**Pull request merged**